DOCS.md

```diff
@@ -354,6 +354,15 @@ bucket
 region
 : AWS bucket region. (`us-east-1`, `eu-west-1`, ...)
 
+account_name
```

Comment:
Let's add prefixes to these. We don't add prefixes to AWS ones to be back-compatible.

Reply:
You actually add the `azure` prefix we just need to update here in docs.